lib/yardstick/v2_client/booking_request.rb

```diff
@@ -20,6 +20,7 @@ def self.process_response(resp, extras = {})
         end
         attrs[:local_start_at] = DateTime.parse(attrs[:local_start_at]) if attrs[:local_start_at].present?
         attrs[:local_end_at] = DateTime.parse(attrs[:local_end_at]) if attrs[:local_end_at].present?
+        attrs[:earliest_date_available] = DateTime.parse(attrs[:earliest_date_available]) if attrs[:earliest_date_available].present?
```

Comment:
Maybe the `earliest_date_available` just needs to wrap itself in a `in_time_zone` call to ensure it's in the proper TZ, then things should work better too.